containerd/containerd.go

```diff
@@ -66,10 +66,18 @@ func (d *Driver) createContainer(image containerd.Image, containerName, containe
 		opts = append(opts, oci.WithPrivileged)
 	}
 
-	// Enable default seccomp profile.
+	if !config.Seccomp && config.SeccompProfile != "" {
```

Comment:
would it make sense to provide a profile and that enables the feature vs doing a bool and what it should implement? This may be how K8s does it and I am not familiar.